js/ui/tab_cookies.js

```diff
@@ -156,6 +156,52 @@
           }
         });
         break;
+      case "whitelist-cookie-domain":
+        var domain = getParentData(element, "data-access");
+        var url = getUrl(domain, "", true);
+        getStorage("cookieWhitelist", function(data) 
+        {
+          let whitelist = data["cookieWhitelist"]
+          if (!whitelist.propertyIsEnumerable(domain)) 
```

Comment:
Detail: I assume you wish to check here if whitelist has "domain" property.

I would suggest to use:
```
if ("domain" in whitelist)
```

This will make code more readable and intention more clear.